src/main/scala/beam/router/skim/ActivitySimSkimmer.scala

```diff
@@ -240,39 +319,49 @@ object ActivitySimSkimmer extends LazyLogging {
     originId: String,
     destinationId: String,
     weightedGeneralizedTime: Double,
+    weightedTotalInVehicleTime: Double,
     weightedGeneralizedCost: Double,
     weightedDistance: Double,
     weightedWalkAccess: Double,
-    weightedWalkEgress: Double,
     weightedWalkAuxiliary: Double,
-    weightedTotalInVehicleTime: Double,
+    weightedWalkEgress: Double,
     weightedDriveTimeInMinutes: Double,
     weightedDriveDistanceInMeters: Double,
-    weightedFerryInVehicleTimeInMinutes: Double,
     weightedLightRailInVehicleTimeInMinutes: Double,
+    weightedFerryInVehicleTimeInMinutes: Double,
     weightedTransitBoardingsCount: Double,
     weightedCost: Double,
     debugText: String = ""
   ) {
 
-    def toCsvString: String = {
-      s"$timePeriodString,$pathType,$originId," +
-      s"$destinationId,$weightedGeneralizedTime,$weightedTotalInVehicleTime," +
-      s"$weightedGeneralizedCost,$weightedDistance,$weightedWalkAccess," +
-      s"$weightedWalkAuxiliary,$weightedWalkEgress,$weightedDriveTimeInMinutes," +
-      s"$weightedDriveDistanceInMeters,$weightedLightRailInVehicleTimeInMinutes,$weightedFerryInVehicleTimeInMinutes," +
-      s"$weightedTransitBoardingsCount,$weightedCost,$debugText\n"
-    }
+    def toCsvString: String = productIterator.mkString("", ",", "\n")
```

Comment:
If moving to utilizing the `productIterator` then we should add a test to make sure any future changes are deliberate.